resources/default_contacts.json

```diff
@@ -18,142 +18,6 @@
-        "dapp?": true,
-        "photo-path": "contacts://name-bazaar"
-    },
-    "cent": {
-        "name" : {"en" : "Cent"},
-        "description": "Get wisdom, get money",
-        "dapp-url": {"en" : "https://beta.cent.co"},
-        "dapp?": true,
-        "photo-path": "contacts://cent"
-    },
-    "erc-dex": {
-        "name" : {"en" : "ERC dEX"},
-        "description": "Trustless trading has arrived on Ethereum",
-        "dapp-url": {"en" : "https://app.ercdex.com/"},
-        "dapp?": true,
-        "photo-path": "contacts://erc-dex"
-    },
-    "oasis-direct": {
-        "name" : {"en" : "Oasis Direct"},
-        "description": "The first decentralized instant exchange",
-        "dapp-url": {"en" : "https://oasis.direct/"},
-        "dapp?": true,
-        "photo-path": "contacts://oasis-direct"
-    },
-    "etherbots": {
-        "name" : {"en" : "Etherbots"},
-        "description": "Robot wars on the Ethereum Platform",
-        "dapp-url": {"en" : "https://etherbots.io/"},
-        "dapp?": true,
-        "photo-path": "contacts://etherbots"
-    },
-    "peepeth": {
-        "name" : {"en" : "Peepeth"},
-        "description": "Blockchain-powered microblogging",
-        "dapp-url": {"en" : "http://peepeth.com/"},
-        "dapp?": true,
-        "photo-path": "contacts://peepeth"
-    },
-    "hexel": {
-        "name" : {"en" : "Hexel"},
-        "description": "Create your own cryptocurrency",
-        "dapp-url": {"en" : "https://www.onhexel.com/"},
-        "dapp?": true,
-        "photo-path": "contacts://hexel"
-    },
-    "kyber": {
-        "name" : {"en" : "Kyber"},
-        "description": "On-chain, instant and liquid platform for exchange and payment service",
-        "dapp-url": {"en" : "https://web3.kyber.network"},
-        "dapp?": true,
-        "photo-path": "contacts://kyber"
-    },
-    "bancor": {
-        "name" : {"en" : "Bancor"},
-        "description": "Bancor is a decentralized liquidity network",
-        "dapp-url": {"en" : "https://www.bancor.network/"},
-        "dapp?": true,
-        "photo-path": "contacts://bancor"
-    },
-    "opensea": {
-        "name" : {"en" : "OpenSea"},
-        "description": "The largest decentralized marketplace for cryptogoods",
-        "dapp-url": {"en" : "https://opensea.io"},
-        "dapp?": true,
-        "photo-path": "contacts://opensea"
-    },
-    "cryptocribs": {
-        "name" : {"en" : "CryptoCribs"},
-        "description": "Travel the globe. Pay in crypto.",
-        "dapp-url": {"en" : "https://cryptocribs.com"},
-        "dapp?": true,
-        "photo-path": "contacts://cryptocribs"
-    },
-    "cryptofighters": {
-        "name" : {"en" : "CryptoFighters"},
-        "description": "Collect train and fight digital fighters.",
-        "dapp-url": {"en" : "https://cryptofighters.io"},
-        "dapp?": true,
-        "photo-path": "contacts://cryptofighters"
-    },
     "simple-dapp": {
```

Comment:
why you didn't move  "simple-dapp"?

Reply:
simple-dapp is not part of the new section based design. Maybe it should be removed from default completely?

Reply:
@rachelhamlin can we add `Developers` sections? or something, simple-dapp is important for testing and developing purposes, it should be visible only in dev mode